gulp-tasks/workbox:reference-docs-v3.js

```diff
@@ -0,0 +1,194 @@
+const gulp = require('gulp');
+const remoteGitTags = require('remote-git-tags');
+const semver = require('semver');
+const fse = require('fs-extra');
+const path = require('path');
+const os = require('os');
+const spawn = require('child_process').spawn;
```

Comment:
in wfHelper.js there's a `function promisedExec(cmd, cwd)` that will does all this for you.